packages/calypso-e2e/src/types/rest-api-client.types.ts

```diff
@@ -26,6 +26,16 @@ export interface NewPostParams {
 	content?: string;
 }
 
+export interface ReaderParams {
+	postID: number;
+	siteID: number;
+}
+
+export interface NewCommentParams {
+	ID: number;
+	content: string;
+}
```

Comment:
**comment**: typically when a new REST API method is implemented in `RestAPIClient`, you'd need to create both the Params and Response versions - just a heads up!

Reply:
Ah, thanks for the heads up! I understand it a lot better now 👍